Server/Server App/Server App/Classes/UserLoginResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Server_App.Classes
+{
+    internal class UserLoginResponse : AbstractResponse
+    {
+        int status;
+
+        public UserLoginResponse(int status)

```

Comment:
The response also needs to contain the balance and the cart for caching at the client-side

Reply:
Made the necessary changes, kindly recheck
@mohanadismail